src/coreclr/jit/register_arg_convention.cpp

```diff
@@ -42,6 +42,17 @@ unsigned InitVarDscInfo::allocRegArg(var_types type, unsigned numRegs /* = 1 */)
         // For System V the reg type counters should be independent.
         nextReg(TYP_INT, numRegs);
         nextReg(TYP_FLOAT, numRegs);
+#elif defined(TARGET_LOONGARCH64)
+        // LA-ABI64.
+        if (numRegs > MAX_PASS_MULTIREG_BYTES / TARGET_POINTER_SIZE)
+        {
+            assert(varTypeIsStruct(type));
+            nextReg(TYP_INT, 1); // TYP_BYREF
+        }                        // TODO:struct-float.
```

Comment:
Change TODO to TODO-LOONGARCH64. Add an NYI or assert if appropriate.

Reply:
OK, Thanks very much.
For now this function is ok for LA and I will delete this comment.